src/BenchmarkDotNet/Toolchains/DotNetCli/DotNetCliGenerator.cs

```diff
@@ -88,5 +99,10 @@ private static bool IsRootSolutionFolder(DirectoryInfo directoryInfo)
             => directoryInfo
                 .GetFileSystemInfos()
                 .Any(fileInfo => fileInfo.Extension == ".sln" || fileInfo.Name == "global.json");
+        
+        private static bool IsRootProjectFolder(DirectoryInfo directoryInfo)
+            => directoryInfo
+                .GetFileSystemInfos()
+                .Any(fileInfo => fileInfo.Extension == ".csproj");
```

Comment:
damn, I have forgotten about F# and VB projects. Could you please handle `.fsproj` and `.vbroj` files as well?

Reply:
@WojciechNagorski ping